demisto_sdk/commands/split/ymlsplitter.py

```diff
@@ -74,6 +75,7 @@ def __init__(self, input: str, output: str = '', file_type: str = '', no_demisto
         self.autocreate_dir = not no_auto_create_dir
         with open(self.input, 'rb') as yml_file:
             self.yml_data = yaml.load(yml_file)
+        self.api_module_path: Optional[str] = None
```

Comment:
Why do you need it?

Reply:
We use it in the contrib-mgmt repo, we want to git add the updated api module path and we get it from the sdk.